camel/storages/vectordb_storages/qdrant.py

```diff
@@ -24,9 +24,178 @@
 from camel.types import VectorDistance
 from camel.utils import dependencies_required
 
+if TYPE_CHECKING:
+    from qdrant_client import QdrantClient
+
 _qdrant_local_client_map: Dict[str, Tuple[Any, int]] = {}
 
 
+class QdrantManager:
```

Comment:
What use cases we must need to have `QdrantManager` separately? What if we just put all things into `QdrantStorage`?